site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -304,16 +320,15 @@ image::tenant-configuration-webhooks-settings.png[Tenant Configuration - JWT,wid
 ===== Table columns
 
 [.api]
-|Event type
-|The event type, this value will be present in the JSON request to identify the message.
-
-[field]#Enabled# [optional]#Optional#::
+Event type::
+The event type, this value will be present in the JSON request to identify the message.
+Enabled::
 When enabled this event can be sent by one or more webhook. You will also need to enable the event for a specific webhook to receive the event.
 +
 This toggle allows you to optionally disable an event for all webhooks all at once.
-
-[field]#Transaction setting# [optional]#Optional#::
+Transaction setting::
 The transaction setting for this event. This setting will apply to all webhooks consuming this event type.
++
 +
```

Comment:
I just corrected a similar issue over in webhooks, see https://github.com/FusionAuth/fusionauth-site/commit/fddb083ae51b9d123a287ad81c3dbeb0d2db9548 for an example. This formatting should be similar where we have an embedded table.